pandas/tests/io/formats/test_style.py

```diff
@@ -1682,6 +1682,11 @@ def f(a, b, styler):
         result = styler.pipe((f, "styler"), a=1, b=2)
         assert result == (1, 2, styler)
 
+    def test_no_cell_ids(self):
+        df = pd.DataFrame(data=[[0]])
```

Comment:
can you add the issue number here as a comment

Reply:
done, still don't know why Linting test fails for these few lines.

Reply:
see https://pandas.pydata.org/docs/dev/development/contributing.html#python-pep8-black